File: Models/BunkerInfo.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace AniDAAPI.Models
{
    public partial class BunkerInfo
    {
        public int BunkerInfoId { get; set; }
        public int CustQuoteMasterId { get; set; }
        public decimal? Hforecd { get; set; }
        public decimal? Mdorecd { get; set; }
        public decimal? GasOilRecd { get; set; }
        public decimal? Fwrecd { get; set; }
        public bool? MsgSent { get; set; }
        public int? LastUpdatedBy { get; set; }
        public DateTime? LastUpdatedDt { get; set; }
    }
}

File: login_and_signup_web-master/lib/Model/BunkerInfo.dart
class BunkerInfo {
  int bunkerInfoID;
  int custQuoteMasterID;
  double hforecd;
  double mdorecd;
  double gasOilRecd;
  double fwrecd;
  // bool msgSent;
  //int lastUpdatedBy;
  //String lastUpdatedDT;

  BunkerInfo({
    this.bunkerInfoID,
    this.custQuoteMasterID,
    this.hforecd,
    this.mdorecd,
    this.gasOilRecd,
    this.fwrecd,
    //this.msgSent,
    //this.lastUpdatedBy,
    //this.lastUpdatedDT
  });

  factory BunkerInfo.fromJson(Map<String, dynamic> json) {
    return BunkerInfo(
      bunkerInfoID: json['bunkerInfoId'],
      custQuoteMasterID: json['custQuoteMasterId'],
      hforecd: json['hforecd'],
      mdorecd: json['mdorecd'],
      gasOilRecd: json['gasOilRecd'],
      fwrecd: json['fwrecd'],
      //    msgSent: json['msgSent'],
      //  lastUpdatedBy: json['lastUpdatedBy'],
      // lastUpdatedDT: json['lastUpdatedDT'],
    );
  }
}
